pkg/version/check.go

```diff
@@ -26,34 +26,29 @@ import (
 
 	"github.com/coreos/go-semver/semver"
 	"github.com/pingcap/kvproto/pkg/metapb"
-	"github.com/pingcap/log"
 	cerror "github.com/pingcap/ticdc/pkg/errors"
 	"github.com/pingcap/ticdc/pkg/httputil"
 	"github.com/pingcap/ticdc/pkg/security"
 	pd "github.com/tikv/pd/client"
-	"go.uber.org/zap"
 )
 
 var (
-	// minPDVersion is the version of the minimal compatible PD.
-	// TODO bump 5.2.0-alpha once PD releases.
-	minPDVersion *semver.Version = semver.New("5.1.0-alpha")
+	// MinPDVersion is the version of the minimal compatible PD.
+	MinPDVersion *semver.Version = semver.New("5.2.0-alpha")
```

Comment:
Can we remove all the version-related global variables? I think we can check whether the major/minor version matched TiKV/PD/TiCDC.